solutions/src/main/java/ru/sberbank/school/task02/ServiceFactoryImpl.java

```diff
@@ -0,0 +1,15 @@
+package ru.sberbank.school.task02;
+
+public class ServiceFactoryImpl implements ServiceFactory {
+
+  @Override
+  public FxConversionService getFxConversionService(ExternalQuotesService externalQuotesService) {
+    return new Converter(externalQuotesService);
```

Comment:
&#39;method def&#39; child has incorrect indentation level 4, expected level should be 8.

*<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>* *<a553732817>*